src/graphql/admin/mongoose_graphql_stats_domain.erl

```diff
@@ -12,6 +12,12 @@
 -include("jlib.hrl").
 
 execute(_Ctx, Domain, <<"registeredUsers">>, _Args) ->
-    {ok, stats_api:stats(<<"registeredusers">>, Domain)};
+    globalStats(<<"registeredusers">>, Domain);
```

Comment:
```suggestion
    domainStats(<<"registeredusers">>, Domain);
```

Reply:
I think the function name "globalStats" here is a bit misleading, because the stats called here are domain stats.